examples/sveltekit-pwa/pwa-configuration.js

```diff
@@ -0,0 +1,91 @@
+const pwaConfiguration = {
+	srcDir: './build',
+	outDir: './.svelte-kit/output/client',
+	mode: 'development',
+	includeManifestIcons: false,
+	scope: '/',
+	base: '/',
+	manifest: {
+		short_name: "PWA Router",
+		name: "PWA Router",
+		start_url: "/",
+		scope: "/",
+		display: "standalone",
+		theme_color: "#ffffff",
+		background_color: "#ffffff",
+		icons: [
+			{
+				src: '/pwa-192x192.png',
+				sizes: '192x192',
+				type: 'image/png',
+			},
+			{
+				src: '/pwa-512x512.png',
+				sizes: '512x512',
+				type: 'image/png',
+			},
+			{
+				src: '/pwa-512x512.png',
+				sizes: '512x512',
+				type: 'image/png',
+				purpose: 'any maskable',
+			},
+		]
+	},
+}
+
+const claims = process.env.CLAIMS === 'true'
+const reload = process.env.RELOAD_SW === 'true'
+const sw = process.env.SW === 'true'
+const replaceOptions = {
+	__DATE__: new Date().toISOString(),
+	__RELOAD_SW__: reload ? 'true' : 'false',
+}
+
+const workboxOrInjectManifestEntry = {
+	// vite and SvelteKit are not aligned: pwa plugin will use /\.[a-f0-9]{8}\./ by default: #164 optimize workbox work
+	dontCacheBustURLsMatching: /-[a-f0-9]{8}\./,
+	globDirectory: './build/',
+	globPatterns: ['**/*.{js,css,html,ico,png,svg,webmanifest}'],
+	globIgnores: sw ? (claims ? ['**/claims-sw*'] : ['**/prompt-sw*']) : ['**/sw*', '**/workbox-*'],
+	// Before generating the service worker, manifestTransforms entry will allow us to transform the resulting precache manifest.
+	// The entries received in the callback from workbox-build module, will contain all the assets specified on srcDir option with the url and its corresponding revision calculated (hash).
+	// Since SvelteKit uses the router name of the directory for all the generated pages, we add a callback to modify the url for all pages.
+	manifestTransforms: [async(entries) => {
```

Comment:
Now that there are docs for this option, we probably don't need as much explanation here.
```suggestion
	// Before generating the service worker, manifestTransforms entry will allow us to transform the resulting precache manifest. See the manifestTransforms docs for mode details.
	manifestTransforms: [async(entries) => {
```